source/Engine.cpp

```diff
@@ -942,8 +942,23 @@ void Engine::Draw() const
 		if(messagePoint.Y() < messageBox.Top())
 			break;
 		float alpha = (it->step + 1000 - step) * .001f;
-		Color color(alpha, 0.f);
-		messageLine.Draw(messagePoint, color);
+		const Color* color;
+		// This switch will make the compiler force us to handle any new importance levels we add in future
+		switch (it->importance) {
+			case Messages::Importance::Highest:
+				color = GameData::Colors().Get("message importance highest");
+				break;
+			case Messages::Importance::High:
+				color = GameData::Colors().Get("message importance high");
+				break;
+			case Messages::Importance::Low:
+				color = GameData::Colors().Get("message importance low");
+				break;
+		}
+		if (color == nullptr) {
+			color = GameData::Colors().Get("messages importance default"); // Will not be null
+		}
```

Comment:
Typo + styleguide corrections:
```suggestion
		if(!color)
			color = GameData::Colors().Get("message importance default");
```

Reply:
Done.